<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectLevel</class>
 <widget class="QDialog" name="SelectLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-image: url(:/Main/Resources/Main/Head.png);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>10</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font: 75 22pt &quot;微软雅黑&quot;;
}</string>
        </property>
        <property name="text">
         <string>选择游戏难度</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(209, 0, 0);</string>
        </property>
        <property name="text">
         <string>行列数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_cell_number">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
	font: 75 16pt &quot;微软雅黑&quot;;
}</string>
        </property>
        <item>
         <property name="text">
          <string>08 X 08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 X 20</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(209, 0, 0);</string>
        </property>
        <property name="text">
         <string>图片数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_image_number">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	font: 75 16pt &quot;微软雅黑&quot;;
}</string>
        </property>
        <property name="text">
         <string>20</string>
        </property>
        <property name="placeholderText">
         <string>10&lt;= N &lt;=50</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(209, 0, 0);</string>
        </property>
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_time">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	font: 75 16pt &quot;微软雅黑&quot;;
}</string>
        </property>
        <property name="text">
         <string>120</string>
        </property>
        <property name="placeholderText">
         <string>20&lt;= T &lt;=600</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(58, 176, 0);
	border-radius: 39px;
	font: 18px &quot;微软雅黑&quot;;
}
QPushButton::pressed{
    padding-left: 2px;
    padding-top: 2px;
	background-color: rgb(40, 121, 59);
}

</string>
        </property>
        <property name="text">
         <string>开始游戏</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(200, 0, 0);
	border-radius: 39px;
	font: 18px &quot;微软雅黑&quot;;
}
QPushButton::pressed{
    padding-left: 2px;
    padding-top: 2px;
	background-color: rgb(145, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
